src/aics-image-viewer/components/App/types.ts

```diff
@@ -1,8 +1,8 @@
-import { View3d, Volume, RawArrayInfo, RawArrayData } from "@aics/volume-viewer";
-import { ImageType, RenderMode, ViewMode } from "../../shared/enums";
-import { PerAxis, MetadataRecord } from "../../shared/types";
-import { ColorArray } from "../../shared/utils/colorRepresentations";
-import { ChannelGrouping, ChannelState, ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
+import type { RawArrayData, RawArrayInfo, Volume } from "@aics/volume-viewer";
+
+import type { MetadataRecord } from "../../shared/types";
+import type { ViewerChannelSettings } from "../../shared/utils/viewerChannelSettings";
+import type { ViewerState } from "../ViewerStateProvider/types";
```

Comment:
thumbs up for using `import type` when it's a type... do we have tsconfig rules to catch this?

Reply:
not that I know of, but looks like it can be done: https://stackoverflow.com/questions/71080256/is-there-a-way-to-automatically-fix-import-type-errors-on-typescript-when-usin